gradle-plugins/src/main/kotlin/io.opentelemetry.instrumentation.muzzle-check.gradle.kts

```diff
@@ -38,14 +41,37 @@ val muzzleTooling: Configuration by configurations.creating {
   isCanBeConsumed = false
   isCanBeResolved = true
 }
+
 val muzzleBootstrap: Configuration by configurations.creating {
   isCanBeConsumed = false
   isCanBeResolved = true
 }
 
+val muzzleShadow by tasks.registering(ShadowJar::class) {
+  from(tasks.jar.get().archiveFile.get())
+
+  configurations = listOf(project.configurations.runtimeClasspath.get())
```

Comment:
this is default, no?

Reply:
defaults are only applied to the default "shadowJar" task, see https://github.com/johnrengelman/shadow/issues/108#issuecomment-62418005